### PR TITLE
Fixes system sound being interrupted upon start up

### LINKED ISSUE #1069: Background music stops on app runs
Some strange plugin behavior (versions 0.9.4 and 0.9.5). If some background music plays and i start my app, music stops. And plays again when i closed my app. 
Here list of my permission in AndroidManifest:
```
<uses-permission android:name="android.permission.INTERNET"/>
<uses-permission android:name="android.permission.READ_EXTERNAL_STORAGE"/>
<uses-permission android:name="android.permission.RECORD_AUDIO" />
<uses-permission android:name="android.permission.MODIFY_AUDIO_SETTINGS" />
<uses-permission android:name="android.permission.VIDEO_CAPTURE" />
<uses-permission android:name="android.permission.AUDIO_CAPTURE" />
<uses-permission android:name="android.permission.CAMERA" />
<uses-permission android:name="android.permission.USE_BIOMETRIC"/>
<uses-permission android:name="android.permission.ACCESS_NETWORK_STATE" />
<uses-permission android:name="android.permission.CHANGE_NETWORK_STATE" />
<uses-permission android:name="android.permission.POST_NOTIFICATIONS" />
<uses-feature android:name="android.hardware.camera"/>
<uses-feature android:name="android.hardware.camera.autofocus" />
```

Steps to reproduce (in my case):
1. Play some streaming music. 
2. Hide music application.
3. Run app with flutter_webrtc plugin (at this step music stops)
4. Close app with flutter_webrtc plugin (at this step music plays again)

* **Flutter version**:
```
[✓] Flutter (Channel beta, 3.1.0-9.0.pre, on macOS 12.5.1 21G83 darwin-arm, locale ru-RU)
    • Flutter version 3.1.0-9.0.pre at /Users/dmitryfomin/Development/FlutterProjects/flutter
    • Upstream repository https://github.com/flutter/flutter.git
    • Framework revision f28e570c8c (3 months ago), 2022-06-14 13:39:33 -0500
    • Engine revision 74ee6b5afd
    • Dart version 2.18.0 (build 2.18.0-165.1.beta)
    • DevTools version 2.14.0

[✓] Android toolchain - develop for Android devices (Android SDK version 30.0.3)
    • Android SDK at /Users/dmitryfomin/Library/Android/sdk
    • Platform android-33, build-tools 30.0.3
    • Java binary at: /Applications/Android Studio.app/Contents/jre/Contents/Home/bin/java
    • Java version OpenJDK Runtime Environment (build 11.0.11+0-b60-7772763)
    • All Android licenses accepted.

[✓] Xcode - develop for iOS and macOS (Xcode 13.4.1)
    • Xcode at /Applications/Xcode.app/Contents/Developer
    • CocoaPods version 1.11.3

[✓] Chrome - develop for the web
    • Chrome at /Applications/Google Chrome.app/Contents/MacOS/Google Chrome

[✓] Android Studio (version 2021.1)
    • Android Studio at /Applications/Android Studio.app/Contents
    • Flutter plugin can be installed from:
      🔨 https://plugins.jetbrains.com/plugin/9212-flutter
    • Dart plugin can be installed from:
      🔨 https://plugins.jetbrains.com/plugin/6351-dart
    • Java version OpenJDK Runtime Environment (build 11.0.11+0-b60-7772763)

[✓] Connected device (3 available)
    some devices

[✓] HTTP Host Availability
    • All required HTTP hosts are available

• No issues found!

```
* **Plugin version**: 0.9.5
* **OS**: Android/IOS

### PR DESCRIPTION
**Pull request by amantoux**
Fixes #1069

### CODE REVIEW

File: android/src/main/java/com/cloudwebrtc/webrtc/audio/AudioSwitchManager.java [hunk 1]
```diff
@@ -65,10 +65,13 @@ public void start() {
                         preferredDeviceList
                 );
                 audioSwitch.start(audioDeviceChangeListener);
-                audioSwitch.activate();
```
Comment (amantoux) [inline android/src/main/java/com/cloudwebrtc/webrtc/audio/AudioSwitchManager.java:68]: This calls `AudioDeviceManager.setAudioFocus()` which causes the interruption

File: android/src/main/java/com/cloudwebrtc/webrtc/MethodCallHandlerImpl.java [hunk 2]
```diff
@@ -988,14 +988,14 @@ public String peerConnectionInit(ConstraintsMap configuration, ConstraintsMap co
...
-        //audioSwitchManager.start();
+        audioSwitchManager.activate();
       } else {
         ArrayList<String> permissions = new ArrayList<>();
         permissions.add(Manifest.permission.BLUETOOTH_CONNECT);
         requestPermissions(
                 permissions,
                 (args) -> {
-                  //audioSwitchManager.start();
+                  audioSwitchManager.activate();
```
Comment (amantoux) [inline android/src/main/java/com/cloudwebrtc/webrtc/MethodCallHandlerImpl.java:998]: Activate sound upon init of peer connection

File: common/darwin/Classes/AudioUtils.m [hunk 3]
```diff
@@ -74,7 +74,6 @@ + (void)setSpeakerphoneOn:(BOOL)enable {
                   :
                   AVAudioSessionCategoryOptionAllowBluetooth | AVAudioSessionCategoryOptionAllowBluetoothA2DP
                     error:nil];
-    [session setActive:YES error:nil];
```
Comment (amantoux) [inline common/darwin/Classes/AudioUtils.m:77]: This causes the interruption (called in `initWithChannel`)
Calling `setActive` seems redundant as the audio session would have already been activated
Did some tests and `setSpeakerOn` seems to work fine without this line

**Comment by amantoux** [comment]
> @cloudwebrtc can you please take a look?
> 
> many thanks

We should maybe wait for https://github.com/flutter-webrtc/flutter-webrtc/pull/1097 to land in `main`

**Review by cloudwebrtc — APPROVED** [review]
lgtm

**Pull request merged**